agent/structs/acl.go

```diff
@@ -45,41 +45,68 @@ const (
 
 	// This policy gives unlimited access to everything. Users
 	// may rename if desired but cannot delete or modify the rules.
-	ACLPolicyGlobalManagementID = "00000000-0000-0000-0000-000000000001"
-	ACLPolicyGlobalManagement   = `
-acl = "write"
+	ACLPolicyGlobalManagementID   = "00000000-0000-0000-0000-000000000001"
+	ACLPolicyGlobalManagementName = "global-management"
```

Comment:
Is there a reason why we have different names for these two builtin policies that are treated the same?
- `global-management`
- `builtin/global-read-only`

I couldn't find a justification for this in the RFC and it feels weird to me that these are named differently.

Reply:
There was a lot of discussion around this one as well. @lornasong?

Reply:
Sure, here's the [section](https://docs.google.com/document/d/1Gp-Up79Q9UAe-ft00Xbva83jAXFU5FH7CklFqCck_XI/edit#bookmark=id.2zc32cx8xn9l) with details naming of `builtin/global-read-only`. But a couple relevant bullets:
 - policy names have to be unique so we wanted to pick a unique name that didn't collide with customers' existing names
 - introducing "/" in policy names prevent colliding with preexisting custom policies
 - "builtin/" prefix aligns with same envoy precedence and was picked to align with how policy templates will move forward with builtin policy templates

Agree that the mis-aligned naming isn't ideal. I think in the long-term `global-management` would become `builtin/global-management` but that would be a breaking change that required a deprecation period. Let us know what you think!

Reply:
:+1: I completely agree with the rationale for the `builtin/` prefix. Ideally I'd like to see it extended to the `global-management` policy as well but I understand that it would be a breaking change have to be handled accordingly. Marking this as resolved.